examples/AdvancedConsumer/AdvancedConsumer.csproj

```diff
@@ -1,7 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Authors>Andreas Heider;Confluent Inc</Authors>
+    <Authors>Andreas Heider;Confluent Inc.</Authors>
+    <Description>Example demonstrating how to consume messages.</Description>
+    <Copyright>Copyright 2016-2017 Confluent Inc., Andreas Heider</Copyright>
+    <PackageTags>Kafka;Confluent;librdkafka</PackageTags>
```

Comment:
I also took out Authors (which is redundant as it's also included in the copyright notice in the source files).